frontend/src/components/uam/components/groups-table/groups-table.tsx

```diff
@@ -10,9 +12,18 @@ type Props = {
 };
 
 const GroupsTable: FC<Props> = ({ columns, data }: Props) => {
+  const navigate = useNavigate();
+
+  const onAddGroupClick = (): void => {
```

Comment:
Please check [A8/A9](https://github.com/BinaryStudioAcademy/quality-criteria/blob/production/source/javascript.md)
```suggestion
  const handleAddGroupClick = (): void => {
```